src/main/java/domain/Store.java

```diff
@@ -0,0 +1,50 @@
+package domain;
+
+import java.util.Objects;
+
+public class Store {
+    private static final String PRICE_NULL_ERROR_MESSAGE = "error : 로또 금액은 Null 값을 입력할수 없습니다.";
+    private static final String PRICE_MIN_ERROR_MESSAGE = "error : 로또 최소가격은 1000원 입니다.";
+    private static final int LOTTERY_PRICE_MIN = 1000;
+    private final int price;
+    private final int lotteryCount;
+
+    public Store(int price) {
+        checkPriceNull(price);
+        validPriceMin(price);
+        this.price = price;
+        this.lotteryCount = calculateNumber();
+    }
```

Comment:
```suggestion
    private Store(int price, int lotteryCount) {
        this.price = price;
        this.lotteryCount = lotteryCount;
    }
```

정의되어 있는 상태값을 모두 전달 받는 생성자를 선언하는것이 어떨까요 > ?